spec/requests/api/v0/events_request_spec.rb

```diff
@@ -139,19 +139,16 @@
     let(:agent) { 'agent' }
     let(:ip_address) { '1.1.1.1' }
 
-    let(:avro_turf) {
-      double({encode: nil})
-    }
+    let(:avro_turf) { KafkaAvroTurf.instance }
```

Comment:
Removed the stubbing of AvroTurf, which exposed a bunch of errors.  Better for us to see them than for Tom to do so :-)